sharedbuffers/mapped_struct.py

```diff
@@ -2803,7 +2803,7 @@ def pack_into(cls, obj, buf, offs, idmap = None, implicit_offs = 0):
                 widmap.link(objid, obj)
 
         packer = cls.PACKER
-        timestamp = int(calendar.timegm(obj.timetuple()))
+        timestamp = int(time.mktime(obj.timetuple()))
```

Comment:
I think it needs to be timegm. There was a case that mktime didn't handle correctly.

Reply:
This is for the `datetime` (that we talked about leaving it as it was). This change was just a "rollback". `date` now uses `calendar.timegm`.

Reply:
No worries! :raised_hands: